components/RunCard.renderCell.tsx

```diff
@@ -169,7 +169,7 @@ function renderMessageWithEmbeddedLinks(result: Result, message: string) {
 					if (isNaN(id as any)) return id // `id` is a URI string
 
 					// Else `id` is a number
-					// TODO: search other location coolections
+					// TODO: search other location collections
```

Comment:
I'm kinda curious about `collections` as there doesn't seem to be any other instance of the term...